tests/integration/new_relations/test_new_relations.py

```diff
@@ -563,7 +563,7 @@ async def test_invalid_extra_user_roles(ops_test: OpsTest):
             f"{DATABASE_APP_NAME}:database", f"{DATA_INTEGRATOR_APP_NAME}:postgresql"
         )
         await ops_test.model.wait_for_idle(apps=[DATABASE_APP_NAME])
-        ops_test.model.block_until(
+        await ops_test.model.block_until(
```

Comment:
There was a missing await here, hiding a bug in the charm behavior :( thankfully @dragomirp caught this typo when looking at the logs when debugging the Nextcloud error.